packages/cli/src/modules/insights/insights.service.ts

```diff
@@ -31,23 +33,31 @@ export class InsightsService {
 		private readonly insightsByPeriodRepository: InsightsByPeriodRepository,
 		private readonly compactionService: InsightsCompactionService,
 		private readonly collectionService: InsightsCollectionService,
+		private readonly pruningService: InsightsPruningService,
```

Comment:
Now that we have two pruning services, maybe we should rename the first one to `ExecutionsPruningService`.

Reply:
Good idea, done !